playbook/app/pb_kits/playbook/pb_icon/docs/_icon_default.jsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react'
+import React from "react"
 
-import Icon from '../_icon'
+import Icon from "../_icon"
+import {Caption} from "../.."
 
 const IconDefault = (props) => {
   return (
     <div>
-      <Icon
-          fixedWidth
-          icon="user"
+      <Icon fixedWidth
+          icon='angles-down'
+          {...props}
```

Comment:
This test shows a single alias working. 
Alias: angles-down
Icon rendered: angle-double-down